qa/integration-tests/src/test/java/io/camunda/it/utils/CamundaMultiDBExtension.java

```diff
@@ -137,12 +138,20 @@ public class CamundaMultiDBExtension
     implements AfterAllCallback, BeforeAllCallback, ParameterResolver {
   public static final String PROP_CAMUNDA_IT_DATABASE_TYPE =
       "test.integration.camunda.database.type";
+  public static final String TEST_INTEGRATION_OPENSEARCH_AWS_URL =
+      "test.integration.opensearch.aws.url";
+  public static final String PROP_TEST_INTEGRATION_OPENSEARCH_AWS_TIMEOUT =
+      "test.integration.opensearch.aws.timeout.seconds";
+  public static final Duration TIMEOUT_DATABASE_EXPORTER_READINESS_TIMEOUT =
+      Optional.ofNullable(System.getProperty(PROP_TEST_INTEGRATION_OPENSEARCH_AWS_TIMEOUT))
+          .map(val -> Duration.ofSeconds(Long.parseLong(val)))
+          .orElse(Duration.ofSeconds(30));
```

Comment:
❓ Should we use here the other static value? TIMEOUT_DATABASE_EXPORTER_READINESS

Reply:
That one is used for other time outs. Would you like to unify those as well? The `TIMEOUT_DATABASE_EXPORTER_READINESS` is 3 minutes, while tests using this value work fine with 30 seconds timeout. My suggestion is not to reuse that constant. If the person set 3 minutes, it is likely for a reason.

Reply:
Don't you set it to 3 minutes in this pr? 🤔

Reply:
Sorry, maybe we're talking on different matters: we have a new timeout that I introduced: `TIMEOUT_DATABASE_EXPORTER_READINESS_TIMEOUT` aka `TIMEOUT_DATA_AVAILABILITY` whish is 30 seconds by default. And we have old ones `TIMEOUT_DATABASE_EXPORTER_READINESS` and `TIMEOUT_DATABASE_READINESS` which are 3 minutes now. Those are used in different places.

The reason why I increased those - I didn't want to add additional parametrized value.

Reply:
BTW, infra team increased instance class for AWS OS. So I will actually decrease timeouts again and see if it is any better now.

Reply:
There's now `m5.large.search` if you're curious